### PR TITLE
QOL-8595 fix DC metadata in page source

### PR DESCRIPTION
**Pull request by ThrawnCA**
- Drop kludge based on detecting ckanext-data-qld controller, instead copy the block into specific templates as required.

### CODE REVIEW

File: ckanext/data_qld_theme/templates/package/read.html [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 {% block meta %}
 {{ super() }}
-{% if c.pkg_dict['organization'] and c.pkg_dict['organization']['title'] %}
-  <meta name="DCTERMS.creator" content="c=AU; o=The State of Queensland; ou={{ c.pkg_dict['organization']['title'] }}" scheme="AGLSTERMS.GOLD" />
+{% if pkg['organization'] and pkg['organization']['title'] %}
```
Comment (duttonw) [inline ckanext/data_qld_theme/templates/package/read.html:5]: Is this inside resource_read that sets

```{% set pkg = c.pkg_dict or pkg_dict %}```

Comment (ThrawnCA) [reply ckanext/data_qld_theme/templates/package/read.html:5]: Actually it comes from the `ckan_extends` directive at the top, which makes the template extend the equivalent core template. Core `ckan/templates/package/read.html` starts with:

        {% set pkg = c.pkg_dict %}

File: ckanext/data_qld_theme/templates/package/resource_read.html [hunk 2]
```diff
@@ -1,25 +1,18 @@
...
-    {% set dataset = h.dataset_display_name(pkg) %}
-    {% if pkg.organization %}
-      {% set organization = h.get_translated(pkg.organization, 'title') or pkg.organization.name %}
-      {% set group_type = pkg.organization.type %}
-      <li>{% link_for _('Organizations'), named_route='organization.index', named_route=group_type + '_index' %}</li>
-      <li>{% link_for organization|truncate(30), named_route='organization_read', id=pkg.organization.name, named_route=group_type + '_read' %}</li>
-    {% else %}
-      <li>{% link_for _('Datasets'), named_route='package.search' %}</li>
-    {% endif %}
-    <li{{ self.breadcrumb_content_selected() }}>{% link_for dataset|truncate(30), named_route='dataset_read', id=pkg.name %}</li>
```
[the remark this answers is not in the document]
Comment (ThrawnCA) [reply ckanext/data_qld_theme/templates/package/resource_read.html:17]: It comes from the parent, ultimately `ckan/templates/package/base.html`. Note the `super()` call that replaces this (now line 15).

File: .ahoy.yml [hunk 3]
```diff
@@ -121,6 +121,7 @@ commands:
       ahoy start-ckan-job-workers
       ahoy start-mailmock &
       sleep 5 &&
+      ahoy cli "behave ${*:-test/features}" --tags @smoke && \
       ahoy cli "behave ${*:-test/features}" || \
```
Comment (duttonw) [inline .ahoy.yml:125]: do you want to exclude smoke from 2nd run? --tags 'not @smoke' https://behave.readthedocs.io/en/latest/tag_expressions.html

**Review by duttonw — APPROVED** [review]

File: .ahoy.yml (hunk 3 above)
Comment (ThrawnCA) [reply .ahoy.yml:125]: We could exclude them, but there's little benefit. They don't take long.

**Pull request merged**